### PR TITLE
D3D9 code comments improvements

### PR DESCRIPTION
**Pull request by K0bin**
I hope those are genuinely a bit useful and not just describing the obvious.

@WinterSnowfall pls review.

### CODE REVIEW

File: src/d3d9/d3d9_device.h [hunk 1]
```diff
@@ -729,10 +732,17 @@ namespace dxvk {
             UINT                    Face,
             UINT                    MipLevel);
 
+    /**
+     * \brief Uploads the given texture subresource from it's local system memory copy.
```
Comment (WinterSnowfall) [inline src/d3d9/d3d9_device.h:736]: Typo

File: src/d3d9/d3d9_device.h [hunk 2]
```diff
@@ -752,6 +762,9 @@ namespace dxvk {
             void**                  ppbData,
             DWORD                   Flags);
 
+    /**
+     * \brief Uploads the given buffer from it's local system memory copy.
```
Comment (WinterSnowfall) [inline src/d3d9/d3d9_device.h:766]: Typo

File: src/d3d9/d3d9_device.h [hunk 3]
```diff
@@ -1105,6 +1187,9 @@ namespace dxvk {
       return (vertexCount - 1) * stride + std::max(m_state.vertexDecl->GetSize(0), stride);
     }
 
+    /**
+     * \brief Writes data to the given pointer and zero any access buffer space
```
Comment (WinterSnowfall) [inline src/d3d9/d3d9_device.h:1191]: "zeroes", if you want to keep it accusative

File: src/d3d9/d3d9_device.cpp [hunk 4]
```diff
@@ -756,7 +766,9 @@ namespace dxvk {
       const Com<D3D9TextureCube> texture = new D3D9TextureCube(this, &desc);
       m_initializer->InitTexture(texture->GetCommonTexture());
       *ppCubeTexture = texture.ref();
-      
+
+
```
Comment (WinterSnowfall) [inline src/d3d9/d3d9_device.cpp:770]: Extra line?

File: src/d3d9/d3d9_device.h [hunk 5]
```diff
@@ -729,10 +732,17 @@ namespace dxvk {
...
+    /**
+     * \brief Uploads the given texture subresource from it's local system memory copy.
+     */
     HRESULT FlushImage(
             D3D9CommonTexture*      pResource,
             UINT                    Subresource);
 
+    /**
+     * \brief Copies the given part of a texture from the local system memory copy of the source texture
+     * to the image of the destination tesxture.
```
Comment (WinterSnowfall) [inline src/d3d9/d3d9_device.h:744]: Typo on the final word.

**Review by WinterSnowfall — CHANGES_REQUESTED** [review]
Some things may be obvious to people familiar with the codebase, but given how much of a quirkfest d3d9 is, I think any and all insights are welcome. It would certainly help the unfortunate souls looking over the code for the first time.

File: src/d3d9/d3d9_device.h (hunk 1 above)
Comment (K0bin) [reply src/d3d9/d3d9_device.h:736]: What about it? I don't see it.

Comment (K0bin) [reply src/d3d9/d3d9_device.h:736]: nvm

**Review by WinterSnowfall — APPROVED** [review]

**Pull request merged**